MUXSDKStats/MUXSDKStats/MUXSDKStats.m

```diff
@@ -176,6 +168,10 @@ + (MUXSDKPlayerBinding *_Nullable)monitorAVPlayerViewController:(nonnull AVPlaye
         MUXSDKAVPlayerViewControllerBinding *newBinding = [[MUXSDKAVPlayerViewControllerBinding alloc] initWithName:name software:MuxPlayerSoftwareAVPlayerViewController andView:player];
         [newBinding setAutomaticErrorTracking:automaticErrorTracking];
         newBinding.playDispatchDelegate = _playerBindingManager;
+        if (domain != nil && domain.length > 0) {
```

Comment:
Can we log an error message if the domain is invalid?

Reply:
also, if they do not set a custom collection domain we want to continue to use litix.io - I might be missing it but where does that happen right now?

Reply:
Oh, it happens in Core-  okay that makes sense https://github.com/muxinc/mux-stats-sdk-objc/pull/119/files#diff-1a15f01bbb046d01e3718e1e9506e210bd76412b3e8b85872e064e69266f9a82R54

Reply:
regarding the "invalid" domain bit, we don't do any domain validation in either StatsSDK or CoreSDK (which matches the behaviour of the android & web SDKs). Do we want to add validation in CoreSDK? Steph brought this up too in the CoreSDK PR

Reply:
the error is now being logged in CoreSDK